app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+      <p class="title">FHIR Bundle</p>
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">FHIR Bundle</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+            <div class="content">
+              <%= @health_card.to_json %>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+    <p class="title">Minified FHIR Bundle</p>
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Minified FHIR Bundle</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <div class="content">
+           <pre>
+              <%= @health_card.bundle.to_json %>
+           </pre>
+          </div>
+        </div>
+      </div>
+    <br>
+    <br>
+    <br>
+
+    <p class="title">Encoded JWS</p>
```

Comment:
The titles don't need to be repeated here and as card headers.

Reply:
@pmadria is this actually resolved? It looks like the title is still repeated

![Screen Shot 2021-05-14 at 11 13 19 AM](https://user-images.githubusercontent.com/41651655/118291312-762a0180-b4a5-11eb-9609-5fb8ddd7563d.png)